siteorigin-panels.php

```diff
@@ -663,6 +663,12 @@ static function get_widget_instance( $class_or_hash ) {
 	 */
 	public function activation_redirect( $plugin ){
 		if( $plugin == plugin_basename( __FILE__ ) ) {
+
+			// Prevent redirect in certain situations
+			if ( wp_doing_ajax() || is_network_admin() || isset( $_GET['activate-multi'] ) ) {
+				return false;
```

Comment:
This shouldn't be returning anything. It's an action handler, not a filter.

Reply:
Oh whoops. Fixing now.

Reply:
Just actually fixed this.